packages/web/src/services/audius-backend/Tipping.ts

```diff
@@ -1,11 +1,14 @@
-import { Supporter, Supporting } from 'common/models/Tipping'
+import { Supporter, Supporting, UserTip } from 'common/models/Tipping'
 import { waitForLibsInit } from 'services/audius-backend/eagerLoadUtils'
 
 export const TIPPING_SUPPORT_DEFAULT_LIMIT = 25
 
 // @ts-ignore
 const libs = () => window.audiusLibs
 
+// todo: I think we should revisit our UserMetadata, User, APIUser models
+// seems a bit off now and kind of confusing, reason why I have any types below
```

Comment:
Not sure exactly what we mean here - imo when we're checking in todos they should be fairly concrete, ideally things we can track as issues